### DISCUSSION TITLE
想问下社交账号的那个图标是怎么处理的

### DISCUSSION THREAD

**Discussion by chitucao** [imsyy/home#69] [Announcements]
我想加个leetcode的主页，但是那个图标的png我看是处理成黑白透明的，不知道咋个处理

**Comment by imsyy** [comment]
我是从 [iconfont](https://www.iconfont.cn/) 上找的，选成白色下载就行

![image](https://user-images.githubusercontent.com/42232682/216742186-64ef5a9f-ac9d-4d14-96cd-704ff05903aa.png)
